src/main/java/frc/robot/Robot.java

```diff
@@ -118,5 +121,17 @@ public void testPeriodic() {
 
   private void manualControl() {
     drive.arcadeDrive(controls.getLeftY(DEADZONE), controls.getRightX(DEADZONE) *0.75);
+    //System.out.println("verticalSpeed" + controls.getLeftY(DEADZONE));
+   // System.out.println("horizontalSpeed" + controls.getRightX(DEADZONE));
+   // System.out.println("LeftEncoder" + drive.leftEncoder.getDistance());
+   // System.out.println("RightEncoder"+ drive.rightEncoder.getDistance());
```

Comment:
Let's not merge commented out code. I encourage you to use Oblog (if you want these values) to get these on the dashboard and get rid of this code.